src/Avalonia.Labs.ExpressionBuilder/ExpressionNodes/ExpressionNode.cs

```diff
@@ -154,7 +154,7 @@ public void Dispose()
 
             if (ExpressionAnimation != null)
             {
-                (ExpressionAnimation as IDisposable)?.Dispose();
+                //(ExpressionAnimation)?.();
```

Comment:
```suggestion
                (ExpressionAnimation as IDisposable)?.Dispose();
```

This probably is an unwanted change.